src/pluto/format/open.cr

```diff
@@ -0,0 +1,9 @@
+module Pluto::Format::Open
```

Comment:
I put this into the `Format` namespace even though it behaves more like an `Operator`, since it directly uses the other formats. Let me know if this is an issue.

Provides a nice convenience method for opening up images.

Reply:
Let's not add these convenience methods as they are simple enough to implement yourself and otherwise just increase complexity of the library.

Reply:
I'm assuming the most common way of creating `Image` objects with this library will be through file reading and writing files. I'd rather that common complexity be taken on by the library than forcing all users of the library to re-implement the same logic. Would it be alright if we kept this and the `Save` modules?

I can remove the specific `open_ppm` and `open_jpeg` methods and handle the `File.open` stuff in this module instead?

Reply:
The thing is, I'm afraid it will only bring clutter because:

1. There are multiple ways to do the same thing. When a user sees the public API, they are presented with a bad choice: to use the convenience methods or not. What if they decide not to? It sounds like a mess.
2. You still have to use `IO` in some cases. For example, in one of my projects, images come from [grok.cr](https://github.com/phenopolis/grok.cr), so should I use `IO` directly in this case and the convenience methods in others? Again, this sounds like a mess.
3. We have to remember to update these methods each time a new format is added to the library.

Reply:
Alternatively, you can create an extension and we will add a link to it in `README.md`.

Reply:
Removed the `Open` and `Save` operators and methods.

I'm hoping your arguments are against a fictional list of poorly implemented "convenience" methods that clutter up the API, rather than this `open` and `save` methods directly :/ 

1. I'm not sure how presenting a user with the choice of convenience methods leads to a mess - assuming the user is intelligent (especially if they're a developer), they'll use the method that most closely matches their use case. It wouldn't be a "bad" choice, it would be "their" choice.
2. For your grok.cr use case, I'd expect you to use whichever method matches your use case the best. If you were reading or writing files primarily, these `open` and `save` methods would probably be a good convenience for you. If you were reading / writing to a socket directly, the `IO` based methods would be what you'd use. Again, I'm not seeing the mess for this specific set of methods.
3. Sure, but there's always a risk of introducing bugs when adding a new format. The type of bug for missing to add this to the convenience methods is a pretty low risk / low consequence bug and easily fixed through a PR.

Regardless, I removed the methods, so there is no concern for scope creep this way.